tests/unit/controllers/account/billing-info/invoices-test.js

```diff
@@ -0,0 +1,12 @@
+import { module, test } from 'qunit';
+import { setupTest } from 'ember-qunit';
+
+module('Unit | Controller | account/billing-info/invoices', function(hooks) {
+  setupTest(hooks);
+
+  // Replace this with your real tests.
```

Comment:
This comment is not really needed

Reply:
Yeah, will remove this test. It got generated automatically when I used the ember CLI to generate the controller.